lib/ClangImporter/ClangAdapter.cpp

```diff
@@ -81,7 +81,7 @@ const clang::Decl *
 importer::getFirstNonLocalDecl(const clang::Decl *D) {
   D = D->getCanonicalDecl();
   auto iter = llvm::find_if(D->redecls(), [](const clang::Decl *next) -> bool {
-    return !next->isLexicallyWithinFunctionOrMethod();
+    return !next->isInLocalScope();
```

Comment:
This one isn't particularly obvious; I assume that is how the upstream Clang sources changed, too?

Reply:
Here is the change: https://github.com/apple/llvm-project/commit/f43859a099fa3587123717be941fa63ba8d0d4f2

`isInLocalScope` is the new name of the method. The implementation did change, though.